helm_deploy/hmpps-welcome-people-into-prison-ui/values.yaml

```diff
@@ -14,6 +14,7 @@ generic-service:
     host: app-hostname.local    # override per environment
     tlsSecretName: hmpps-welcome-to-prison-cert
     path: /
+    external-dns.alpha.kubernetes.io/aws-weight: "100"
```

Comment:
this needs to be inside an annotations section

Reply:
Upgraded to 1.1.2 version